addons/common/XEH_PREP.hpp

```diff
@@ -15,6 +15,7 @@ PREP(getInventory);
 PREP(getPhoneticName);
 PREP(getPlayers);
 PREP(getPosFromScreen);
+PREP(getReflectorSelections);
```

Comment:
In my opinion, getLightingHitpoints or something similar is quicker to understand than this.